tests/test.yml

```diff
@@ -0,0 +1,92 @@
+- name: Run test profile
+  command: nextflow run main.nf -profile test,docker --citest true --round_DE 3
+  tags:
+    - test
+  files:
+    - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_genotype_WT_vs_KO.tsv
+      md5sum: 93fd663cb976c102144e1ffb34d88e53
+    - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_treatment_Treated_vs_Control.tsv
+      md5sum: cb1f49e7fe93b4ef2673eb99fd8793cd
+    - path: results_test/differential_gene_expression/final_gene_table/final_DE_gene_list.tsv
+      md5sum: 81d3437334849b3b4de4de6729a3d54d
+    - path: results_test/differential_gene_expression/gene_counts_tables/deseq2_table.tsv
```

Comment:
Shouldn't all tsv files be reproducible and output the same md5sum? Why are the md5sums missing for all these tsv files?

Reply:
Unfortunately, for some tsv files, the 5sums are not consistent, so I had to remove them...